src/locales/pl.ts

```diff
@@ -50,335 +50,344 @@ export default {
+    mary_ursula_of_jesus_ledochowska_virgin: 'Św. Urszuli Ledóchowskiej, dziewicy',
+    vincent_of_saragossa_deacon: 'Św. Wincentego, diakona i męczennika',
+    vincent_de_paul_priest: 'Św. Wincentego à Paulo, prezbitera',
+    vincent_ferrer_priest: 'Św. Wincentego Ferreriusza, prezbitera',
+    vincent_pallotti_priest: 'Św. Wincentego Pallottiego, prezbitera',
+    wenceslaus_i_of_bohemia_martyr: 'Św. Wacława, męczennika',
+    zdislava_of_lemberk: 'Św. Zdzisławy',
+    zygmunt_gorazdowski_priest: 'Św. Zygmunta Gorazdowskiego, prezbitera',
+    sigmund_felix_felinski_bishop: 'Św. Zygmunta Szczęsnego Felińskiego, biskupa',
+    seven_holy_founders_of_the_servite_order: 'Świętych siedmiu założycieli Zakonu Serwitów Najświętszej Maryi Panny',
+    seven_martyred_nuns_from_the_franciscan_missionaries_of_mary: 'Siedmiu męcznnic ze Zgromadzenia Franciszkanek Misjonarek Maryi',
+    passion_of_saint_john_the_baptist: 'Męczeństwo św. Jana Chrzciciela',
+    most_holy_name_of_jesus: 'Najświętszego Imienia Jezus',
+    visitation_of_mary: 'Nawiedzenie Najświętszej Maryi Panny',
+    our_lady_of_sorrows_principal_patroness_of_slovakia: 'Najświętszej Maryi Panny Bolesnej, głównej patronki Słowacji',
+    george_of_lydda_martyr_principal_patron_of_england: 'Św. Grzegorza, męczennika i głównego patrona Anglii',
+    benedict_of_nursia_abbot_principal_patron_of_europe: 'Św. Benedykta, opata i głównego patrona Europy',
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_secondary_patrons_of_europe: 'Świętych Cyryla, mnicha i Metodego, biskupa, patronów Europy',
+    teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe: 'Św. Teresy Benedykty od Krzyża, dziewicy, męczennicy i patrona Europy',  // TODO: How to say `secundary patroness` in Polish?
+    andrew_apostle_principal_patron_of_scotland: 'Św. Andrzeja, Apostoła i głównego patrona Szkocji',
+    bridget_of_sweden_religious_secondary_patroness_of_europe: 'Św. Brygidy, zakonnicy i patronki Europy',  // TODO: How to say `secundary patroness` in Polish?
+    catherine_of_siena_virgin_secundary_patroness_of_europe: 'Św. Katarzyny Sieneńskiej, dziewicy, doktora Kościoła i patronki Europy',  // TODO: How to say `secundary patroness` in Polish?
+    catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe: 'Św. Katarzyny Sieneńskiej, dziewicy, doktora Kościoła i patronki Włoch i Europy',  // TODO: How to say `secundary patroness` in Polish?
+    patrick_of_ireland_bishop_principal_patron_of_ireland: 'Św. Patryka, biskupa i głównego patrona Irlandii',
+    columba_of_iona_abbot_secondary_patron_of_ireland: 'Św. Kolumbana, zakonnika i patrona Irlandii',  // TODO: How to say `secundary patron` in Polish?
+    our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Najświętszej Maryi Panny z Guadalupe, głównej Patronki obu Ameryk',
+    francis_of_assisi_principal_patron_of_italy: 'Św. Franciszka z Asyżu, głównego patrona Włoch',
```

Comment:
@jarosz, could you proof the Polish patronage titles please? Below I list them all. There are also some other issue which you might be able to fix. Thanks in advance. :smiley:

```js
adalbert_of_prague_bishop: 'Św. Wojciecha, biskupa i męczennika, głównego patrona Polski',
peter_liu_wenyuan_martyr: 'Św. Piotra Liu, Męczennika',  // TODO: Change to `Saint Peter Liu Wenyuan, Martyr`
peter_wu_guosheng_martyr: 'Św. Piotra Wu, Męczennika',  // TODO: Change to `Saint Peter Wu Guosheng, Martyr`
stanislaus_of_szczepanow_bishop: 'Św. Stanisława, biskupa i męczennika, głównego patrona Polski',
stanislaus_kostka_religious: 'Św. Stanisława Kostki, zakonnika, patrona Polski',
teresa_of_jesus_of_avila_virgin: 'Św. Teresy od Jezusa, dziewicy i doktora Kościoła',  // TODO: Localise as `Saint Teresa of Jesus of Ávila, Virgin and Doctor of the Church`
our_lady_of_sorrows_principal_patroness_of_slovakia: 'Najświętszej Maryi Panny Bolesnej, głównej patronki Słowacji',
george_of_lydda_martyr_principal_patron_of_england: 'Św. Grzegorza, męczennika i głównego patrona Anglii',
benedict_of_nursia_abbot_principal_patron_of_europe: 'Św. Benedykta, opata i głównego patrona Europy',
cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_secondary_patrons_of_europe: 'Świętych Cyryla, mnicha i Metodego, biskupa, patronów Europy',
teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe: 'Św. Teresy Benedykty od Krzyża, dziewicy, męczennicy i patrona Europy',  // TODO: How to say `secundary patroness` in Polish?
andrew_apostle_principal_patron_of_scotland: 'Św. Andrzeja, Apostoła i głównego patrona Szkocji',
bridget_of_sweden_religious_secondary_patroness_of_europe: 'Św. Brygidy, zakonnicy i patronki Europy',  // TODO: How to say `secundary patroness` in Polish?
catherine_of_siena_virgin_secundary_patroness_of_europe: 'Św. Katarzyny Sieneńskiej, dziewicy, doktora Kościoła i patronki Europy',  // TODO: How to say `secundary patroness` in Polish?
catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe: 'Św. Katarzyny Sieneńskiej, dziewicy, doktora Kościoła i patronki Włoch i Europy',  // TODO: How to say `secundary patroness` in Polish?
patrick_of_ireland_bishop_principal_patron_of_ireland: 'Św. Patryka, biskupa i głównego patrona Irlandii',
columba_of_iona_abbot_secondary_patron_of_ireland: 'Św. Kolumbana, zakonnika i patrona Irlandii',  // TODO: How to say `secundary patron` in Polish?
our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Najświętszej Maryi Panny z Guadalupe, głównej Patronki obu Ameryk',
francis_of_assisi_principal_patron_of_italy: 'Św. Franciszka z Asyżu, głównego patrona Włoch',
```